### PR TITLE
Add evt::callEvent

### PR DESCRIPTION
**Pull request by Makonede**

### CODE REVIEW

File: src/build.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def run_make(self):
         self.context.log("Dumping symbols from elf...")
         with open(elf_path_to_syms(self.context.elf), "w+", encoding="utf-8") as syms:
             result = subprocess.run([
-                "objdump",
+                "aarch64-none-elf-objdump",
```
Comment (Pistonight) [inline src/build.py:106]: Does this change things?

Comment (Makonede) [reply src/build.py:106]: hey, sorry, this fork is kind of a mess rn and the listing isn't currently working. that line itself allows it to work on windows, where `objdump` doesn't exist by default. please don't merge this pr right now, i'll let you know when it's ready

Comment (Pistonight) [reply src/build.py:106]: hmm this tool is kinda meant for linux only because windows is kinda weird

Comment (Pistonight) [reply src/build.py:106]: maybe this can be made a config

Comment (Makonede) [reply src/build.py:106]: the tool works fine on windows aside from that particular part it seems, and `aarch64-none-elf-objdump` seems like a better option than `objdump` on linux as well and it should exist on both platforms as long as you have devkitA64

**Pull request closed without merging**